app/lib/searching.rb

```diff
@@ -0,0 +1,62 @@
+class Searching
+  require 'gds_api/rummager'
+  attr_reader :params
+  def initialize(params)
+    @params = params
+  end
+
+  def count
+    return 10 if params["count"].blank?
+    return 1000 if params["count"].to_i > 1000
+    params["count"]
+  end
+
+  def call
+    terms = params["search_term"].split.join("+")
```

Comment:
It's a bit of a throwback to before the API gem was introduced. I needed it then to build the URL in a recognisable way. I'll check to see if it works without now, good spot :)